docs/src/pages/discover-more/roadmap/roadmap.md

```diff
@@ -34,6 +34,8 @@ Our GitHub project's roadmap is where you can learn about what features we're wo
 - [MUI X](https://github.com/mui-org/material-ui-x/projects/1).
   This repository focuses on providing advanced React components.
   Some of the features are MIT licensed, others are available under a commercial license.
+- [MUI Design kits](https://github.com/mui-org/mui-design-kits/projects/1)
+  This repository focuses on providing the components for designers on Figma, Sketch, and Adobe XD.
```

Comment:
```suggestion
  This repository focuses on gathering feedback and requests for the components available on Figma, Sketch, and Adobe XD.
```

Clearing out a bit more. The repo doesn't provide the actual components - they're in the design tool. Rather, the repo gathers feedback from the users of the Kits.

Reply:
We talked about using the repository to host the Figma plugin and documentation.

Reply:
Got it. Yeah, I think that a repo focused on the plug-in is better, for sure. They are different products: the Kits and the plug-in.